### PR TITLE
Fix few broken links

### PR DESCRIPTION
**Pull request by jorge07**

### CODE REVIEW

File: content/en/docs/aws/customizing-aws.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ nodeGroups:
 
 ### Customize Authentication
 
-If you use {{% config-uri-aws-standard %}}, you can consider to change the default password in the [configuration file]({{% config-uri-aws-standard %}}). The configuration file contains:
+If you use [configuration file]({{% config-uri-aws-standard %}}), you can consider to change the default password in the [configuration file]({{% config-uri-aws-standard %}}). The configuration file contains:
```
Comment (8bitmp3) [inline content/en/docs/aws/customizing-aws.md:71]:
```suggestion
If you use the [configuration file]({{% config-uri-aws-standard %}}), you can consider to change the default password in the [configuration file]({{% config-uri-aws-standard %}}). The configuration file contains:
```

File: content/en/docs/aws/customizing-aws.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ spec:
     username: admin@kubeflow.org
 ```
 
-If you use {{% config-uri-aws-cognito %}}, please see [this section](/docs/aws/authentication)
+If you use [configuration file with cognito]({{% config-uri-aws-cognito %}}), please see [this section](/docs/aws/authentication)
```
Comment (8bitmp3) [inline content/en/docs/aws/customizing-aws.md:81]:
```suggestion
If you use the [configuration file with cognito]({{% config-uri-aws-cognito %}}), please see [this section](/docs/aws/authentication)
```

File: content/en/docs/aws/customizing-aws.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ spec:
...
 
 ### Customize IAM Role for Pods
 Please see [this section](/docs/aws/iam-for-sa)
```
Comment (8bitmp3) [inline content/en/docs/aws/customizing-aws.md:84]:
```suggestion

Refer to the [Setting up AWS IAM role for Service Account](/docs/aws/iam-for-sa) guide.
```

("see" -> "refer to" or "go to" for accessibility)
(use full name of guides for better UX)

File: content/en/docs/aws/customizing-aws.md [hunk 4]
```diff
@@ -87,4 +87,4 @@ Please see [this section](/docs/aws/iam-for-sa)
 Please see [this section](/docs/aws/private-access)
```
Comment (8bitmp3) [inline content/en/docs/aws/customizing-aws.md:87]:
```suggestion

Check the [How to create private EKS clusters](/docs/aws/private-access) guide.
```

("see" -> "refer to" or "go to" for accessibility)
(use full name of guides for better UX)

File: content/en/docs/aws/customizing-aws.md [hunk 5]
```diff
@@ -87,4 +87,4 @@ Please see [this section](/docs/aws/iam-for-sa)
...
 
 ### Customize Logging
-Please see [this section](/docs/aws/logging)
\ No newline at end of file
+Please see [this section](/docs/aws/logging)
```
Comment (8bitmp3) [inline content/en/docs/aws/customizing-aws.md:90]:
```suggestion

Go to the [Add logging support for Kubeflow](/docs/aws/logging) guide for more information.
```

("see" -> "refer to" or "go to" for accessibility)
(use full name of guides for better UX)

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Good stuff @joerg84, I checked https://www.kubeflow.org/docs/aws/customizing-aws/ and the link is indeed in need of a fix.

This is after: 
image

Preview:

https://deploy-preview-2377--competent-brattain-de2d6d.netlify.app/

**Comment by 8bitmp3** [comment]
Thanks @jorge07 👍 

/assign @Jeffwan @PatrickXYS 

PTAL. Thank you!

**Comment by PatrickXYS** [comment]
@jorge07 Thanks for contribution!

Overall lgtm

/lgtm
/approve

**Pull request merged**